pkg/azureupdate/validate_azureclusterconfig_test.go

```diff
@@ -280,8 +280,8 @@ func getClusterConfigAdmissionRequest(oldVersion string, newVersion string) *v1b
 			Kind:    "AzureClusterUpgrade",
 		},
 		Resource: metav1.GroupVersionResource{
-			Version:  "provider.giantswarm.io/v1alpha1",
-			Resource: "azureconfigs",
+			Version:  "core.giantswarm.io/v1alpha1",
+			Resource: "azureclusterconfigs",
```

Comment:
HmmMmm... why is this needed?

Reply:
it's unrelated, but it was an error in master I wanted to fix